### ISSUE TITLE
Windows cert check breaks for non-English locale

### ISSUE THREAD

**Issue by tresf** [qzind/tray#229]
On Windows, QZ Tray uses a command [`certutil -store Root`](https://github.com/qzind/tray/blob/4218d630208074e91054e8079edc88603217c6f9/ant/windows/windows-keygen.js.in#L580) to determine if a certificate has been installed properly.

This works well for English locales but fails on non-english locales and may not report a certificate being installed properly.

This is due to the [use of the English word `Issuer:`](https://github.com/qzind/tray/blob/4218d630208074e91054e8079edc88603217c6f9/ant/windows/windows-keygen.js.in#L621) which may be different on a non-English locale.

For example, when French is the default locale, `Issuer:` is replaced with `Émetteur:` and the check fails.  The software is usable, but an [undesirable and incorrect](https://user-images.githubusercontent.com/6345473/29994385-77953d64-8f9c-11e7-908e-77f9b3bd2e33.jpg) warning appears stating it was unsuccessful.

**Comment by bobo3k** [comment]
possible to do kind of a regex to just search for the string "QZ Industries"? 
that could help bypass the issue...

Here is what it looks like in french:

```
================ Certificat 5 ================
Numéro de série : 7c9f1bea
Émetteur: CN=localhost, E=support@qz.io, OU=QZ Industries, LLC, O=QZ Industries, LLC, L=Canastota, S=NY, C=US
 NotBefore : 02/09/2017 10:39
 NotAfter : 02/09/2037 10:39
Objet: CN=localhost, E=support@qz.io, OU=QZ Industries, LLC, O=QZ Industries, LLC, L=Canastota, S=NY, C=US
La signature correspond à la clé publique
Certificat racine : le sujet correspond à l’émetteur
Hach. cert. (sha1) : 877c8c34fa2ef876413cfaab459f59a2ba98c808
Aucune information sur le fournisseur de clé
Impossible de trouver le certificat et la clé privée pour le déchiffrement.
```

**Comment by tresf** [comment]
This problem [exists on Unix too](https://github.com/qzind/tray/blob/2.0/src/qz/utils/ShellUtilities.java#L48).  The cleanest way when scraping the output is to force English locale.

**Comment by bobo3k** [comment]
In windows 10 it seems to be a pain though... just searching for ways to get the certutil command results other than in french, but so far, no luck....

**Comment by tresf** [comment]
The technique (f679b54) makes some assumptions...
1. The values are on a specific line of the cert block.
2.  The values come after a description and a colon delimiter.

If these assumptions become incorrect in a future Windows release or in a certain language, the logic will break and the Windows XP dialog will appear.

**Comment by bobo3k** [comment]
Hi Tres, 
just confirming that it works perfectly on my french windows 10.
No warning, it goes through without any issue... (I sent you the setup logs by email).

B

**Comment by bobo3k** [comment]
I can now confirm that this setup works on french computers & spanish (just tried it on a spanish w10)!

Good job!

B

**Comment by tresf** [comment]
@bobo3k thanks for the feedback.  Closed via f679b54.  This fix will be available in 2.0.5.  Please feel free to use the `v2.0.4-lang` until 2.0.5 is released.
